documentation/CAMARA-Security-Interoperability.md

```diff
@@ -62,6 +63,36 @@ Unless otherwise noted, all the protocol parameter names and values are case sen
 ### Transport Security
 All network connections MUST use TLS 1.2 or better.
 
+### Sender-Constrained Tokens
+
+[OAuth 2.0 Security Best Current Practice](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-security-topics#section-2.2.1) and the [FAPI 2.0 Baseline Profile](https://openid.net/specs/fapi-2_0-baseline-01.html) both RECOMMEND that authorization and resource servers use mechanisms for sender-constraining access tokens. 
+
+This document states that Demonstrating Proof of Possession (DPoP) [RFC9449](https://datatracker.ietf.org/doc/html/rfc9449) MAY be used by API Consumers, to prevent misuse of stolen and leaked access tokens. 
+
+CAMARA authorization servers MUST NOT respond with an error if they do not support DPoP. DPoP allows authorization servers to issue tokens that are not sender-constrained even if a valid DPoP header is present in the authorization request. It is up to the API consumer to decide whether none-sender-constrained tokens meet their security requirements.
+
+If the API Provider supports DPoP, support for it MAY be expressed by the server metadata field `dpop_signing_alg_values_supported` or alternate API documentation.
+If the API Provider supports DPoP, the API Consumer MUST use DPoP.
```

Comment:
```suggestion
```
There is disagreement about `If the API Provider supports DPoP, the API Consumer MUST use DPoP.` as per https://github.com/camaraproject/IdentityAndConsentManagement/pull/225#issuecomment-2520153353 and https://github.com/camaraproject/IdentityAndConsentManagement/pull/225#issuecomment-2531036735. This feedback has not yet been addressed.

CC @eric-murray